integration/js/utils/WebdriverSauceLabs.js

```diff
@@ -203,6 +203,9 @@ class SaucelabsSession {
       .withCapabilities(cap)
       .forBrowser(capabilities.browserName)
       .build();
+    const timeout = 600000;
```

Comment:
Should we only do it for iOS?

Reply:
We could, but I think it would be better to set it for everything. Selenium has a default timeout for all browsers, which could in theory change at some point, so it would be better to just make it uniform across the board

Reply:
Ok looked a bit more into it, and the default script timeout is supposed to be 30 seconds across the board, previously this was not the case. For IOS Safari it's undefined for some reason. I couldn't find a bug report for it, but I changed the code so now it will only override the timeout if it's undefined or less then 30 seconds.